### ISSUE TITLE
bug: After adding collection-repeat for list native scrolling is not happening 

### ISSUE THREAD

**Issue by anushkamahesh** [driftyco/ionic#3783]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>android</span> <span ionic-platform-version>4.4</span>  <span ionic-webview>webview</span>

<span ionic-description>I have changed my ng-repeat list to collection-repeat due to rendering issue in long list. But after adding that the native scrolling is gone. It is not that much smooth scrolling now. Previously it was there with the ng-repeat.

Thank you  </span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @anushkamahesh!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3783). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3783)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by perrygovier** [comment]
It's not every day I can reply to an issue with a ryhme:
http://blog.ionic.io/native-scrolling-in-ionic-a-tale-in-rhyme/

TLDR:
This is by design. `collection-repeat` by nature of what it does requires most of what JS scrolling already does. Then when you make components like `ion-item` work inside of collection-repeat, you wind up with a near exact clone of JS scrolling. So we just set it so that if you use collection-repeat, it forces the page to use JS scrolling. 

TLDR for the TLDR:
`cake != eat cake`

**Comment by tbro028** [comment]
Is there any way to have a collection-repeat for IOS and native for android? I set global property for android to native but collection-repeat will overwrite that...?

Native scrolling doesn't appear to work in ios7 iphone 4... or not in my app anyway...

**Comment by Monca-xu** [comment]
@perrygovier 
i want to make collection and native together, but i find that when i scroll, easy to get stuck, and must manually trigger the onScroll
